.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,34 @@
 <!---
+
 Thanks for creating a pull request!
 
-Please start your PR title with the appropriate prefix as specified by [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#specification):
+## TODO
```

Comment:
```suggestion
Please, make sure you addressed the following points before marking your PR as ready for review.
```

Reply:
I made this a heading for @arctic-alpaca's point about making it easy to leave in uncompleted steps in a draft PR, so you can just delete the comment markers and the completed steps.